augur/application/db/models/augur_operations.py

```diff
@@ -739,8 +739,9 @@ class UserRepo(Base):
         ForeignKey("augur_data.repo.repo_id", name="user_repo_user_id_fkey"), primary_key=True, nullable=False
     )
 
-    repo = relationship("Repo")
-    group = relationship("UserGroup")
+    #made redundant by user_repo relationship in Repo orm class.
+    #repo = relationship("Repo")
+    #group = relationship("UserGroup")
```

Comment:
These are not redundant. `repo = relationship("Repo")` still needs to be present on the UserRepo class to enable to ability to do UserRepo.repo and get back the related Repo object. Placing a relationship on the UserRepo relationship on the Repo class has no effect on the UserRepo class.